ruby/anagram.rb

```diff
@@ -0,0 +1,16 @@
+class Anagram
+  attr_reader :root_word
+
+  def initialize(root_word)
+    @root_word = root_word
+  end
+
+  def match(candidates)
+    anagrams = []
+    for c in candidates do
```

Comment:
Is there any specific reason you are using a `for/in` loop? (Hint: look up the difference between `for/in` and `each`)